README.md

```diff
@@ -57,9 +57,9 @@ You can generate abbreviations from help text using
 
 ## Comparison to Other Tools
 
-Contextual abbreviations work like [globalias][glob] or [zsh-abbr][abbr],
-except that they are *contextual* - `a` expands differently when it comes after
-*`git` vs. `docker`.
+Contextual abbreviations work like [globalias][glob]
+except that they are *contextual* (`a` expands differently when it comes after
+*`git` vs. `docker`), and like [zsh-abbr][abbr] but without having to manually configure permutations and with support for interleaving other text (defining `git restore`'s `s` and `p` confers support for `git restore s p`, `git restore p s`, `git restore --quiet s p`, `git restore s --quiet p`, etc.).
```

Comment:
This sentence is now quite long. Perhaps the comparison to `zsh-abbr` could be moved into its own paragraph? Additionally, this is a bit confusing to read - I'm not completely sure which tool requires "manually configuring permutations" or which tool has "support for interleaving other text". Perhaps this could be made more clear by explicitly contrasting the behavior of `zsh-abbr` with that of `zsh-contextual-abbrevs` for these different examples?

Reply:
Ok, I might close this for now until another zsh-abbr enthusiast comes along to help -- Feel free to file an issue tracking improving this comparison. Thanks again!